There is some kind of a linting error, but the linter doesn't really say what's the problem 

```
$ /home/runner/work/content/content/node_modules/.bin/markdownlint-cli2 --fix files/en-us/web/api/videoencoder/configure/index.md files/en-us/web/api/videoencoder/encode/index.md
markdownlint-cli2 v0.10.0 (markdownlint v0.31.1)
Finding: files/en-us/web/api/videoencoder/configure/index.md files/en-us/web/api/videoencoder/encode/index.md !node_modules !.git !.github !tests
Linting: 2 file(s)
Summary: 0 error(s)
Done in 0.[42](https://github.com/mdn/content/actions/runs/6334274236/job/17227847037?pr=29332#step:12:43)s.

Logs from front-matter linter:
0 0

Please fix all the linting issues mentioned in above logs and in the review comments.
Error: Process completed with exit code 1.
```

--- a/files/en-us/web/api/videoencoder/configure/index.md
+++ b/files/en-us/web/api/videoencoder/configure/index.md
@@ -47,15 +47,15 @@ configure(config)
         - `"keep"`
     - `scalabilityMode`
       - : A string containing an encoding scalability mode identifier as defined in [WebRTC](https://w3c.github.io/webrtc-svc/#scalabilitymodes*).
-    - `bitrateMode`
+    - `bitrateMode` {{optional_inline}}
       - : A string containing a bitrate mode. One of:
         - `"constant"`
-          - : The encode will target constant bitrate.
+          - : The encoder will target constant bitrate.
         - `"variable"` (default)
-          - : The encode will target a variable bitrate, allowing more space to be used for complex signals and less space for less complex signals.
+          - : The encoder will target a variable bitrate, allowing more space to be used for complex signals and less space for less complex signals.
         - `"quantizer"`
-          - : The encoder will disregard `bitrate` field and instead it will use a codec specific quantizer values specified for each frame in {{domxref("VideoEncoder.encode()")}} options.
-    - `latencyMode`
+          - : The encoder will disregard the `bitrate` option and instead it will use codec-specific quantizer values specified for each frame in the `options` parameter to {{domxref("VideoEncoder.encode()")}}.
+    - `latencyMode` {{optional_inline}}
       - : A string containing a value that configures the latency behavior of this codec. One of:
         - `"quality"` (default)
           - : The encoder should optimize for encoding quality.

--- a/files/en-us/web/api/videoencoder/encode/index.md
+++ b/files/en-us/web/api/videoencoder/encode/index.md
@@ -23,18 +23,18 @@ encode(frame, options)
 - `frame`
   - : A {{domxref("VideoFrame")}} object.
 - `options` {{optional_inline}}
-  - : An object containing the following member:
-    - `keyFrame`
+  - : An object containing the following members:
+    - `keyFrame` {{optional_inline}}
       - : A {{jsxref("boolean")}}, defaulting to `false` giving the user agent flexibility to decide if this frame should be encoded as a key frame. If `true` this indicates that the given frame must be encoded as a key frame.
-    - `vp9`
+    - `vp9` {{optional_inline}}
       - : Encode options for the [VP9](/en-US/docs/Web/Media/Formats/Video_codecs#vp9) codec.
         - `quantizer`
-          - : Frame quantizer value 0 to 63
-    - `av1`
+          - : Frame quantizer value 0 to 63.
+    - `av1` {{optional_inline}}
       - : Encode options for the [AV1](/en-US/docs/Web/Media/Formats/Video_codecs#av1) codec.
         - `quantizer`
           - : Frame quantizer value 0 to 51
-    - `avc`
+    - `avc` {{optional_inline}}
       - : Encode options for the [AVC (H.264)](/en-US/docs/Web/Media/Formats/Video_codecs#avc_h.264) codec.
         - `quantizer`
           - : Frame quantizer value 0 to 63
@@ -57,7 +57,7 @@ In the following example `encode` is passed a `VideoFrame`, and the options para
 encoder.encode(frame, { keyFrame: true });
 ```
 
-Setting per-frame QP value for while encoding individual frames.
+Setting per-frame QP value for encoding individual frames.
 
 ```js
 let encode_options = { keyFrame: false };

--- a/files/en-us/web/api/videoencoder/encode/index.md
+++ b/files/en-us/web/api/videoencoder/encode/index.md
@@ -29,15 +29,15 @@ encode(frame, options)
     - `vp9` {{optional_inline}}
       - : Encode options for the [VP9](/en-US/docs/Web/Media/Formats/Video_codecs#vp9) codec.
         - `quantizer`
-          - : Frame quantizer value 0 to 63.
+          - : Frame quantizer value 0 to 63. Only effective if {{domxref("VideoEncoder")}} was configured with `quantizer` bitrate mode.
     - `av1` {{optional_inline}}
       - : Encode options for the [AV1](/en-US/docs/Web/Media/Formats/Video_codecs#av1) codec.
         - `quantizer`
-          - : Frame quantizer value 0 to 51
+          - : Frame quantizer value 0 to 63. Only effective if {{domxref("VideoEncoder")}} was configured with `quantizer` bitrate mode.
     - `avc` {{optional_inline}}
       - : Encode options for the [AVC (H.264)](/en-US/docs/Web/Media/Formats/Video_codecs#avc_h.264) codec.
         - `quantizer`
-          - : Frame quantizer value 0 to 63
+          - : Frame quantizer value 0 to 51. Only effective if {{domxref("VideoEncoder")}} was configured with `quantizer` bitrate mode.
 ### Return value
 
 None ({{jsxref("undefined")}}).
@@ -60,18 +60,29 @@ encoder.encode(frame, { keyFrame: true });
 Setting per-frame QP value for encoding individual frames.
 
 ```js
-let encode_options = { keyFrame: false };
+const encoder = new VideoEncoder(init);
+const encoderConfig = {
+  codec: 'vp09.00.10.08',
+  width: 800,
+  height: 600,
+  bitrateMode: 'quantizer',
+  framerate: 30,
+  latencyMode: 'realtime'
+};
+encoder.configure(encoderConfig);
+
+const encodeOptions = { keyFrame: false };
 const qp = calculateQp(codec, frame);
 
 if (codec.includes("vp09")) {
-  encode_options.vp9 = { quantizer: qp };
+  encodeOptions.vp9 = { quantizer: qp };
 } else if (codec.includes("av01")) {
-  encode_options.av1 = { quantizer: qp };
+  encodeOptions.av1 = { quantizer: qp };
 } else if (codec.includes("avc")) {
-  encode_options.avc = { quantizer: qp };
+  encodeOptions.avc = { quantizer: qp };
 }
 
-encoder.encode(frame, encode_options);
+encoder.encode(frame, encodeOptions);
 ```
 
 ## Specifications

--- a/files/en-us/web/api/videoencoder/encode/index.md
+++ b/files/en-us/web/api/videoencoder/encode/index.md
@@ -62,12 +62,12 @@ Setting per-frame QP value for encoding individual frames.
 ```js
 const encoder = new VideoEncoder(init);
 const encoderConfig = {
-  codec: 'vp09.00.10.08',
+  codec: "vp09.00.10.08",
   width: 800,
   height: 600,
-  bitrateMode: 'quantizer',
+  bitrateMode: "quantizer",
   framerate: 30,
-  latencyMode: 'realtime'
+  latencyMode: "realtime",
 };
 encoder.configure(encoderConfig);
 

